tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go

```diff
@@ -359,6 +363,7 @@ func NormalizeResourceProviderName(input string) string {
 		"microsoft.windowsesu":                     "Microsoft.WindowsESU",
 		"microsoft.windowsiot":                     "Microsoft.WindowsIoT",
 		"microsoft.workloadmonitor":                "Microsoft.WorkloadMonitor",
+		"PaloAltoNetworks.Cloudngfw":               "PaloAltoNetworks.CloudNGFW",
```

Comment:
The Key here needs to be lower-cased (see the usage below), so this wants to be:

```suggestion
		"paloaltonetworks.cloudngfw":               "PaloAltoNetworks.CloudNGFW",
```

Reply:
Thanks. Updated.